test/runnable/inclusive_incontracts.d

```diff
@@ -0,0 +1,33 @@
+// REQUIRED_ARGS: -preview=inclusivein
+// PERMUTE_ARGS: -O -inline
+
+import core.exception : AssertError;
+import std.exception;
+
+void main()
+{
+    class A
+    {
+        void foo()
+        in (true, "foo")
+        {
+        }
+    }
+
+    class B : A
+    {
+        override void foo()
+        in (false, "nope not foo")
+        {
+        }
+    }
+
```

Comment:
Please add a test using multiple `in` contracts.

Reply:
like this?